frog/imports/client/AccountModal/SignUp.js

```diff
@@ -0,0 +1,210 @@
+// @flow
+import React from 'react';
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import TextField from '@material-ui/core/TextField';
+import Link from '@material-ui/core/Link';
+import Grid from '@material-ui/core/Grid';
+import LockOutlinedIcon from '@material-ui/icons/LockOutlined';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+import { Meteor } from 'meteor/meteor';
+import { withStyles } from '@material-ui/styles';
+import {
+  errorBasedOnChars,
+  emailErrors,
+  passwordErrors
+} from './validationHelpers';
+
+import { type SignUpStateT, type SignUpPropsT } from './types';
+
+const styles = (theme: Object) => ({
```

Comment:
We should use the `makeStyle` function, which has for bonus that the `theme` object gets automatic type inference :tada: 
https://material-ui.com/styles/basics/

Reply:
This would mean I would have to change my component to functional and use hooks. We discussed the reason for me not using hooks. So, therefore I feel we should keep it as it is.